_posts/2020-01-07-jump_dev_2020.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "JuMP-dev 2020: Abstract Submission"
+date:   2020-01-07
+categories: [jump-dev]
+---
+
+June 15-17, 2019, [Louvain-la-Neuve, Belgium](https://goo.gl/maps/fyxqevrKn76nvhwq9).
+
+This workshop is the fourth JuMP-dev workshop after the [first](/meetings/mit2017/),
+[second](/meetings/bordeaux2018/), and [third](/meetings/santiago2019) JuMP-dev workshops.
+
+The purpose of the workshop is to bring together students, researchers, and practitioners
+with interests in the software aspects of [JuMP](https://github.com/JuliaOpt/JuMP.jl) and
+related packages. The workshop is open for anybody to attend. In particular, we invite new
+contributors and those who have not met the core development team.
+
+## Abstract Submission
+
+**[Submit an abstract.](https://docs.google.com/forms/d/e/1FAIpQLSd7d3TJ2de0GSD56lEBrR8e6KmVlPANjwNsr09KroSAuIq-Xw/viewform?usp=sf_link)**
+
+Similar to previous JuMP-devs, we invite participants to (optionally) present work related
+to JuMP. In particular, we especially seek talks related to:
+
+ - the development of new solvers and modeling paradigms (e.g., MINLP, multi-objective)
+ - stories of your success (or failure and why it failed) using JuMP in industy
+ - stories of your success (or failure and why it failed) using JuMP in teaching
+
+Talks should be aimed at a general audience, but familiarity with JuMP/Julia can be assumed.
+
+See the [previous](/meetings/mit2017/) [JuMP-dev](/meetings/bordeaux2018/)
+[workshops](/meetings/santiago2019) for examples of accepted talks.
+
+Abstract submission will close **February 29**, and we expect to notify accepted speakers in
+early March.
+
+## Registration
+
+A registration portal will be created shortly. Unlike previous years, there will be a
+registration fee of approximately USD 50.
```

Comment:
We should finalize the registration fee before we put down a number. For now we can just say that there will be a small registration fee.